docs/overview.md

```diff
@@ -0,0 +1,82 @@
+The Gnosis Multisig is configured as owner of:
+- Wrapped tokens deployed on the EVM chain (f.e WHBAR / WHTS tokens)
+- The Router smart contract
+
+#### Adding / Removing Members
+Validators can add new members or remove members from the validator set. We expect validators to have an off-chain communication channel using which validators can discuss the current setup, vote on removing validators or adding new ones.
+
+Two transactions must be executed in order for the Validator set to change.
+1. On Hedera, `Crypto Update` transaction that modifies the `n/m` threshold accounts (`Bridge` and `Fees`)
+2. On the EVM chain, `updateMember` transaction that modifies the list of members in the `Router` contract (it may add or remove a member)
+
+### Fees
+The main incentive for becoming a Validator is the service fee paid by users. The fee is a percentage of the transferred amount, paid on the native asset.
+For example transferring 100 HBAR from Hedera to the EVM chain (WHBAR) is going to have 1% service fee (1 HBAR) transferred to the Bridge Validators.
+
+*Note: The Service fee is configurable property and determined by the validators*
+
+### Transferring HBAR/HTS from Hedera to the EVM chain
+
+The transfer of assets from Hedera to the EVM chain is described in the following sequence diagram.
+<p align="center">
+  <img src="./assets/hedera-to-evm.png">
+</p>
+
+#### Steps
+1. **Initiating the transfer**
+   Alice wants to transfer `HBAR`/`HTS token` from Hedera to the EVM chain. She opens any UI that integrates the Bridge and sends the asset to the `Bridge` Account. The memo of the transfer contains the `evm-address` which is going to be the receiver of the wrapped asset on the other chain.
+2. **Picking up the Transfer**
+   The Bridge validator nodes listen for new incoming transfers to the `Bridge` Account. Once they pick up the new transaction, they verify the `state proof` and validate that the `memo` contains a valid EVM address configured as receiver of the wrapped asset.
+3. **Paying out fees**
+
+3.1 Each of the Validators create a Schedule Create transaction transferring the `service fee` amount from the `Bridge` account to the list of validators equally *(f.e if the service fee is `7 HBAR` and there are `7` validators, the Schedule Create Transfer will contain Transfer list crediting `1 HBAR` to each of the validators.)*
+3.2 Due to the nature of Scheduled Transactions, only one will be successfully executed, creating a scheduled Entity and all others will fail with `TRANSACTION_ALREADY_SCHEDULED` error, and the transaction receipt will include the `ScheduleID` of the first submitted transaction.
+All validators, except the one that successfully created the Transaction execute `ScheduleSign` and once `n out of m` validators execute the Sign operation, the  transfer of the fees will be executed.
```

Comment:
`the  transfer of the fees are executed` -> `the transfer of the fees are executed`